lms/templates/homework.html

```diff
@@ -9,7 +9,7 @@
 
     <section>
       <input type="hidden" name="problem_id" value="${ problem['name'] }">
-      <input type="submit" value="Check">
+      <input type="submit" value="Submit">
```

Comment:
I am still verifying where this is used.  @cahrens Do you know if I should change this template?